### ISSUE TITLE
Add option to download and upload storage files

### ISSUE THREAD

**Issue by jeromepl** [jeromepl/highlighter#25]
Adding this would allow users to more easily sync their highlights between devices while keeping the extension completely free to use as there would be no need to pay for storage servers

**Comment by GokulNC** [comment]
Indeed, this would a great feature to export the contents of localStorage so that it can be imported in other devices!

**Comment by adamhuang1416gapp** [comment]
Hi 
Thanks for the good work! this is a great extension to use!
The question is: I know how to download my highlights to a json file, but how do I upload it back?
